examples/testmonitor/results/Results.cs

```diff
@@ -0,0 +1,165 @@
+                TotalTimeInSeconds = random.NextDouble() * 10,
+                Parameters = parameters,
+                DataModel = "TestStand",
+            };
+
+            return stepData;
+        }
+
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
+
+            /*
+             * Create the TestDataManager for communicating with the server.
+             */
+            var testDataManager = new TestDataManager(configuration);
+
+            // Intialize the random number generator
+            Random random = new Random();
+
+            // Set test limits
+            var lowLimit = 0;
+            var highLimit = 70;
+
+            // Initialize a ResultData object
+            var resultData = new ResultData()
+            {
+                Operator = "mvaterla",
+                ProgramName = "Power Test",
+                Status = new Status(StatusType.Running),
+                SerialNumber = Guid.NewGuid().ToString(),
+                PartNumber = "NI-ABC-123-PWR",
+                FileIds = new List<string> { }
+            };
+            // Create the test result on the SystemLink server
+            var testResult = testDataManager.CreateResult(resultData);
+            testResult.AutoUpdateTotalTime = true;
+
+            /*
+            * Simulate a sweep across a range of electrical current and voltage.
+            * For each value, calculate the electrical power (P=IV).
+            * Simulate some random current and voltage loss after each test to
+            * provide some randomness the the values.
+            */
+            var current = 0;
+            var voltage = 0;
+            var currentLoss = 1 - random.NextDouble();
+            var voltageLoss = 1 - random.NextDouble();
+            var power = current * currentLoss * voltage * voltageLoss;
+            for (current = 0; current < 10; current++)
+            {
+                currentLoss = 1 - random.NextDouble() * 0.25;
+                power = current * currentLoss * voltage * voltageLoss;
+                // Generate the step data
+                var currentStepData = GenerateStepData($"Current Sweep", "SequenceCall", current, voltage, power, lowLimit, highLimit);
+                // Create the step on the SystemLink server
+                var currentStep = testResult.CreateStep(currentStepData);
+
+                for (voltage = 0; voltage < 10; voltage++)
+                {
+                    voltageLoss = 1 - random.NextDouble() * 0.25;
+                    power = current * currentLoss * voltage * voltageLoss;
+                    // Generate the step data
+                    var voltageStepData = GenerateStepData($"Voltage Sweep", "NumericLimit", current, voltage, power, lowLimit, highLimit);
```

Comment:
Perhaps the child steps should be called something like `Measure Voltage`, since each one of them is a specific test and not a full sweep? Each top level step is a sweep over voltages at one current - is `Voltage Sweep` a better fit for the parent steps?

Reply:
Voltage and Current are the step inputs.  The thing being measured is the power output.

So maybe name the parent step "Voltage Sweep" and the child step "Measure Power Output"?

Reply:
Yeah, I like that.